src/tools/rest-backend/cryptlite/base64.h

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
The idea is to get rid of boost and cryptlite completely.

Why do you need sha256?